web/backend/src/Server.ts

```diff
@@ -26,6 +26,27 @@ const app = express();
 
 app.use(morgan('tiny'));
 
+let enf: Enforcer;
+
+const enforcerLoading = newEnforcer('model.conf', 'policy.csv');
```

Comment:
do you need this to be in the global scope, or could you put it (along with the promise resolution below) in an initialization function ?